app/views/users/show.html.erb

```diff
@@ -1,11 +1,18 @@
-<h1><%= @user.username %>'s Profile Page</h1>
+<h1>Hello <%= @user.username %>!</h1>
 
+<div class="grid-x grid-margin-x">
+    <div class="image-card">
+        <img src="<%= @user.profile_image_url%>">
```

Comment:
not sure where this profile image is coming from, it doesn't look like carrierwave or image uploading has been added to the app yet?

Reply:
We haven't implemented image uploading or carrierwave just yet.